sovrin_client/test/cli/test_nym_suspension.py

```diff
@@ -105,7 +105,7 @@ def testTrusteeSuspendingTrustAnchor(be, do, trusteeCli, trustAnchorAdded,
        expect=nymAddedOut, mapper=trustAnchorAdded)
     s = SimpleSigner().identifier
     be(trustAnchorCli)
-    errorMsg = 'UnauthorizedClientRequest'
+    errorMsg = "UnauthorizedClientRequest('USER cannot add USER'"
```

Comment:
We can, but I think it would be nice to specify why it's not permitted (the reason is that because the person is neither Trustee not Steward nor TA).
And I think it would be nice to have a term for a user with None role in any case